build_tools/github_actions/test_executable_scripts/test_rocroller.py

```diff
@@ -81,6 +81,20 @@
 elif TEST_TYPE == "quick":
     test_filter_arg = "--gtest_filter=*quick*"
 
+# Append to the existing filter or start a negative-only filter
+_excluded = [
+    "AssertTest/GPU_AssertTest.GPU_Assert/28",
+    "AssertTest/GPU_AssertTest.GPU_UnconditionalAssert/28",
+    "AssertTest/GPU_AssertTest.GPU_Assert/29",
+    "AssertTest/GPU_AssertTest.GPU_UnconditionalAssert/29",
+    "GPU_KernelTests/GPU_KernelTest.GPU_WholeKernel/1",
+]
```

Comment:
1. Share the logs for these failing tests (so people don't need to dig through github actions logs that will expire)
2. Notify rocroller developers, ideally by identifying a culprit commit that broke the tests
    * We may need to revert a breaking change that caused the tests to start failing, so we need to get that process started with the right developers notified

We shouldn't exclude tests without those steps at a minimum.